test/promisepay_ex/charges/charge_test.exs

```diff
@@ -0,0 +1,40 @@
+require IEx
+
+defmodule GetChargeTest do
+  use PromisepayEx.ApiCase
+
+  test "charge returns Map (200)" do
+    use_cassette "charges/get/200" do
+      charge = PromisepayEx.charge("f4a701cf-2950-4423-a6bf-604bcf846466")
+
+      assert is_map(charge)
+
+      assert charge.id
+      assert charge.account_id
+      assert charge.amount
+    end
+  end
+
+  test "company raises error (401)" do
+    use_cassette "companies/get/401" do
+      try do
+        PromisepayEx.company("7b85aa9c-fc54-4449-afef-f52fc2b94cd3")
```

Comment:
@psyfear should be `PromisepayEx.charge("f4318485-6291-46de-a331-7e94da56efd5")`?